scripts/west_commands/runners/sctool.py

```diff
@@ -10,6 +10,8 @@
 class SCTOOLBinaryRunner(ZephyrBinaryRunner):
     '''Runner front-end for Senscomm Flash Tool.'''
 
+    DEFAULT_ADDR = '0x80000000'
```

Comment:
Why we can't CONFIG_FLASH_BASE_ADDRESS instead of 0x80000000?

Reply:
now it takes from the config